src/tests_cfg/cake_filling_price.rs

```diff
@@ -35,6 +35,8 @@ pub enum PrimaryKey {
 }
 
 impl PrimaryKeyTrait for PrimaryKey {
+    type ValueType = (i32, i32);
```

Comment:
Seems like we are able to support composite key now!

Reply:
Indeed